lib/nailed/config.rb

```diff
@@ -3,13 +3,23 @@
 #
 module Nailed
 
+  DEFAULT_CONFIG_PATH =
+    File.join(TOPLEVEL, "config", "default-config.yml")
+  CONFIG_PATH =
+    File.join(TOPLEVEL, "config", "config.yml")
   #
   # Config
   #
   class Config
     # load config.yml once
     def self.content
-      @@conf ||= File.join(TOPLEVEL,"config","config.yml")
+      unless @@conf
```

Comment:
removing the memoization actually breaks it here :/ when `@@conf` doesn't exist yet we are running into an error.
I would suggest sth like
```ruby
@@conf ||= DEFAULT_CONFIG_PATH
if File.exist?(CONFIG_PATH)
  @@conf = CONFIG_PATH
end
```